services/core-web/src/components/mine/Permit/PermitConditionForm.spec.tsx

```diff
@@ -4,6 +4,7 @@ import { ReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
 import { PERMITS } from "@mds/common/constants/reducerTypes";
 import * as MOCK from "@mds/common/tests/mocks/dataMocks";
 import PermitConditionForm from "./PermitConditionForm";
+import { PermitConditionsProvider } from "./PermitConditionsContext";
```

Comment:
oh man you read my mind. The props passing down is getting _seriously_ unwieldly!